hypothesis-python/tests/ghostwriter/test_expected_output.py

```diff
@@ -24,19 +24,30 @@
 import operator
 import pathlib
 import re
+import sys
 from typing import Sequence
 
 import numpy
 import pytest
 
+import hypothesis
 from hypothesis.extra import ghostwriter
+from hypothesis.utils.conventions import not_set
 
 
 @pytest.fixture
 def update_recorded_outputs(request):
     return request.config.getoption("--hypothesis-update-outputs")
 
 
+def get_recorded(name, actual=""):
+    name, actual
```

Comment:
It seems like this line doesn't have an effect. Should it be here?

Reply:
Nope, and it's gone 👀